src/lib/messages.ts

```diff
@@ -46,7 +46,13 @@ export const messages = {
         messageGenerator: (severity: number, message: string) => `Sev${severity}: ${message}`,
         source: {
             generator: (engine: string) => `${engine} via Code Analyzer`,
-            isSource: (source: string) => source.endsWith(' via Code Analyzer'),
+            isSource: (source: string) => {
+                if (!source) {
+                    return false;
+                }
+                
+                return source.endsWith(' via Code Analyzer');
+            },
```

Comment:
Lower golf score:

isSource: (source: string) => source && source.endsWith(' via Code Analyzer'),

Reply:
@gbockus-sf - looks like you need code signing set up. I'll take care of this if that's fine with you. Apologies for the delay - this has been on my todo ever since you reported. I should have gotten to this quicker!

Reply:
@gbockus-sf : Here is the [PR](https://github.com/forcedotcom/sfdx-code-analyzer-vscode/pull/119) with similar fix to another function + tests. @stephen-carter-at-sf: Both your suggestions return undefined instead of false when source is falsy in my tests. This was a surprise to me. I have left the code as is for now. I'll go ahead and close this PR without merging.